commit f3095493b06d52a33ae77a0b8c1f92a22633261a (from 116dc86b8a34ffe57b083df554e4ab87cb359b58)
Merge: 116dc86 ce07c91
Author: ErikZalm <erik@vdzalm.eu>
Date:   Mon Mar 5 09:56:50 2012 -0800

    Merge pull request #100 from tommycheng/patch-1
    
    While Waiting for hotbed to warm up via M190, LCD Display and Click Enco...

diff --git a/Marlin/Marlin.pde b/Marlin/Marlin.pde
index e82afab..0a5d5a2 100644
--- a/Marlin/Marlin.pde
+++ b/Marlin/Marlin.pde
@@ -942,6 +942,7 @@ void process_commands()
             codenum = millis(); 
           }
           manage_heater();
+          LCD_STATUS;
         }
         LCD_MESSAGEPGM(MSG_BED_DONE);
         previous_millis_cmd = millis();

